### PR TITLE
docs: add minikube video

### PR DESCRIPTION
**Pull request by cdrage**
docs: add minikube video

### What does this PR do?

* Adds a small video detailing installing from catalog to using the
  minikube creation feature as well as playing a YAML
* Creates a new page on how to install the extension + use it
* Adds the video to the page

### Screenshot / video of UI

Website related, see Netlify preview :)

### What issues does this PR fix or reference?

Closes https://github.com/containers/podman-desktop-internal/issues/367

### How to test this PR?

N/A

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

**Comment by cdrage** [comment]
https://github.com/containers/podman-desktop-media/pull/23 MUST be merged in first in order for this to show.

**Review by Firewall — APPROVED** [review]
👍 Looks great

File: website/docs/kind/installing-extension.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+description: Tutorial on how to install the Minikube extension and use it
+keywords: [podman desktop, podman, minikube, kubernetes]
+tags: [podman, kubernetes, minikube]
+---
+
+import ReactPlayer from 'react-player'
+
+# Installing the Minikube extension
+
+The Minikube extensions provides the capability of creating a local Kubernetes cluster. This extension can be installed and used as a one-click deployment for a test cluster.
```
Comment (shipsing) [inline website/docs/kind/installing-extension.md]:
```suggestion
The Minikube extension provides the capability of creating a local Kubernetes cluster. This extension can be installed and used as a one-click deployment for a test cluster.
```

File: website/docs/kind/installing-extension.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+#### Procedure
+
+If you have Podman Desktop already installed, <a href="podman-desktop:extension/podman-desktop.minikube">**click to launch the installation**</a> of Minikube in Podman Desktop.
+
+If not, please [install Podman Desktop](/docs/installation) first.
+
+#### Tutorial
+
+Follow the below video for a complete installation guide from install to cluster creation:
```
Comment (shipsing) [inline website/docs/kind/installing-extension.md]:
```suggestion
The following video provides a complete guide from installation to cluster creation:
```

**Comment by shipsing** [comment]
@cdrage, the content looks fine. 
A query: Why we placed this new section in the Kind category, not in the Minikube category?
Also, I could not watch the video. The video screen seemed frozen. nothing played. I checked it in the Netlify preview.

image

**Comment by cdrage** [comment]
> @cdrage, the content looks fine. 
> 
> A query: Why we placed this new section in the Kind category, not in the Minikube category?
> 
> Also, I could not watch the video. The video screen seemed frozen. nothing played. I checked it in the Netlify preview.
> 
> 
> 
> image
> 
> 

That's my mistake! Should have been in the other folder. 

I'll update the PR.

**Comment by cdrage** [comment]
@shipsing Fixed! Good catch. Confirm it's in the minikube section + video works now.

Screenshot 2024-10-17 at 8 30 47 AM

**Comment by shipsing** [comment]
LGTM, @cdrage :)
Which tool you used to create the video??

**Comment by cdrage** [comment]
> LGTM, @cdrage :) Which tool you used to create the video??

Just the built-in macOS screenshot tool!

**Pull request merged**